test/performance/shmem_perf_suite/shmem_bibw_atomics_perf.c

```diff
@@ -169,7 +169,7 @@ static inline void bw_set_metric_info_len(perf_metrics_t *metric_info)
                                         sizeof(long long)};
     int snode = streaming_node(*metric_info);
     atomic_op_type op_type = OP_ADD;
-    metric_info->type = BI_DIR;
+    metric_info->b_type = BI_DIR;
```

Comment:
Should this be set prior to the call to `streaming_node`?

Reply:
Looks like `streaming_node` doesn't check this field, but maybe a good idea to switch the order anyway?  Also, it would be good if `streaming_node` called `is_streaming_node` with `my_info.my_node` to eliminate the copy/paste.  Maybe file an issue for that?

Reply:
Yes, `streaming_node` does not depend on this. The only metric it depends on is the `comm_style`. I will reverse the order anyway. 
Thanks for the other suggestion. Will fix it in a future PR.